Update callout Layout Development Guide (#1814)
vtexdocs/dev-portal-content: docs/guides/Integration-Guides/payments-integration-guide/layout-development-guide-for-payment-methods-in-smart-checkout-vtex.md

The following sections show how to create a layout to be used for payment methods in VTEX's Smart Checkout.

> >ℹ️ It is not possible to create a new layout for custom payment methods (those developed by the merchant and that only work in their stores, such as notes payable, co-branded, or private label cards).

Reviewer: ```suggestion
>ℹ️ It is impossible to create a new layout for custom payment methods (those developed by the merchant and that only work in their stores, such as notes payable, co-branded, or private label cards).
```

> > ℹ️ It is not possible to create a new layout to be used in custom payment methods (those developed by the merchant and that only work in their stores, such as notes payable, co-branded or private label cards).

Reviewer: ```suggestion
>ℹ️ It is not possible to create a new layout for custom payment methods (those developed by the merchant and that only work in their stores, such as notes payable, co-branded, or private label cards).
```
